src/components/policy/policy_external/include/policy/cache_manager.h

```diff
@@ -604,12 +604,6 @@ class CacheManager : public CacheManagerInterface {
                    const std::string& wers_country_code,
                    const std::string& language);
 
-  /**
-   * @brief Checks, if specific head unit is present in PT
-   * @return boot Suceess, if present, otherwise - false
-   */
-  bool IsMetaInfoPresent() const;
```

Comment:
@LitvinenkoIra How this removal is related to redundant GetSystemInfo sending?

Reply:
@AByzhynar we call this function **only** inside PolicyManagerImpl::OnSystemReady() and nowhere else.